proxy.go

```diff
@@ -239,7 +239,18 @@ func printAgentLogs(sock string) error {
 	otherMask := 0007
 	other := int(fileInfo.Mode().Perm()) & otherMask
 	if other != 0 {
-		return fmt.Errorf("All socket permissions for 'other' should be disabled, got %3.3o", other)
+		logger().WithField("socket permissions", fmt.Sprintf("%4.4o", fileInfo.Mode().Perm())).
+			Infof("All socket permissions for 'other' should be disabled")
+
+		// Setting permissions socket for "other" to 0.
+		otherMask := 0770
```

Comment:
You could make this `const`

Reply:
This is actually not a declaration. I was trying to reassign a new value to the previously declared variable. 
PS: I have now changed this from a declaration to an assignment.

Reply:
Good point! :smile: